aws-ecs-service-fargate/outputs.tf

```diff
@@ -34,4 +34,9 @@ output "alb_https_listener_arn" {
 output "alb_http_listener_arn" {
   description = "ALB HTTP listener ARN, only if HTTPS forwarding is disabled"
   value       = var.disable_http_redirect ? aws_lb_listener.http.arn : null
-}
\ No newline at end of file
+}
+
+output "ecs_service_arn" {
+  description = "The ARN of the ECS service that is created"
+  value       = aws_ecs_service.id
```

Comment:
this is probably missing something like `aws_ecs_service.<some name>.id`

Reply:
o duh